scalyr_agent/tests/linux_process_metrics_test.py

```diff
@@ -24,6 +24,35 @@
 
 
 class TestMetricClass(ScalyrTestCase):
+    def test_metric_as_object_key(self):
+        dd = {}
```

Comment:
Add a comment here about what the bug was so that folks understand why you are adding a lot of instances of the object.

Reply:
Comment added and whitespace error removed